index.html

```diff
@@ -3,11 +3,67 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-        <title>Hello Microverse</title>
-        <link rel="stylesheet" href="../Hello-Microverse/style.css">
+        <title>Setup and Mobile Version Skeleton</title>
+        <link rel="stylesheet" href="../setup-and-mobile-version-skeleton/style.css">
     </head>
-    <body>
-        <h1>Hello Microverse</h1>
+    <body >
+        <header id="header-container">
+            <div class="logo-container">My logo</div>
```

Comment:
- [ ] [Optional] Kindly put the logo inside an `a` tag.

Reply:
@ShahierNashaat Thank you sir for the review
I will put the logo inside a tag as per your recommendation.